### ISSUE TITLE
Exception thrown generating MDDDL file

### ISSUE THREAD

**Issue by adrianlfns** [empira/MigraDoc#22]
Hello,

 I'm using Migradoc (version 1.50.5147.0) in order to build a PDF document... In my scenario I'm just trying to write the mdddl file. I have the exact code that will throw an exception (See file attached).     

Just as a background... The error only appears when we write a paragraph text that have exactly 193 characters (no more or no less), followed by a new line character...

Please advise.

Thank you

Adrian

[ConsoleApp4.zip](https://github.com/empira/MigraDoc/files/3048355/ConsoleApp4.zip)

**Comment by mokhin-denis** [comment]
A catch this exception with my Document too. The MDDDL file has been created only once, and then I cach an excaption every time. And MDDDL file is creating with different length every time. I tried to use string as filename, I opened Stream by myself - all the same. It works fine only with WriteString, but not WriteDocument. I mean - first copy Document to string (MigraDoc.DocumentObjectModel.IO.DdlWriter.WriteToString), then write string to file
